### DISCUSSION TITLE
등록된 글이 처음에는 노출이 되었다가 다시 안되는 이유는 뭔가요?

### DISCUSSION THREAD

**Discussion by devkuma** [awesome-devblog/awesome-devblog#712] [Q&A] [answered]
안녕하세요. 구독 이메일 관련 질문이 드립니다.
제 블로그를 등록을 하고, 구독 이메일에 2번(11/30, 12/01) 노출이 되고 그 다음부터는 되지를 않네요.
뭐가 노출이 되는 등록되는 기준이 뭐가 따로 있나요? 아님 제 블로그가 블록이라도? ㅜㅜ
https://www.devkuma.com/index.xml

**Comment by sarojaba** [comment, marked as answer]
안녕하세요.
이메일 구독은 http://daily-devblog.com/ 에서 문의 부탁드립니다.

이메일 구독은 별개의 프로젝트이고, 다른 분이 운영하고 계십니다.
awesome-devblog의 블로그 목록만 참고하고 계시고, 수집은 별개로 진행하고 있습니다.
그래서 제가 노출 조건 등은 알 수가 없네요.

**Comment by devkuma** [comment]
친절한 답변 감사합니다. ^^
